skeleton/jest/package.json

```diff
@@ -34,6 +35,7 @@
       "json"
     ],
     "transform": {
+      "^.+\\.(css|less|sass|scss|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "jest-transform-stub",
```

Comment:
Please add ext `styl` as cli supported Stylus.

Reply:
Added styl to transform regex